{{cookiecutter.project_slug}}/.gitlab/README.md

```diff
@@ -0,0 +1,8 @@
+# Gitlab
+
+add the following in your __gitlab-ci.yml__ file:
+
+```yaml
+include:
```

Comment:
thought for later: I guess this could be done with a post_gen_project hook as well.

Reply:
yes, for gitlab i will make a separate round before I release this version.